action_for_github_server/action.yml

```diff
@@ -110,6 +116,7 @@ runs:
           printf '===\nALKiln GitHub server - GitHub will now show the output of docker creation. ALKiln cannot control this output. If you want to prevent output, use "with:" and set the input "SHOW_DOCKER_OUTPUT" to "false". This output will only be visible to those who can already see the action output, like people with admin or write permissions.\n==='
         shell: bash
 
+      # TODO: Reduce these to 1 block by putting ` > /dev/null 2>&1` into an env var
       # No output
       - name: "ALKiln GitHub server - Download and start docker silently"
         if: ${{ env.OUTPUT_DOCKER == 'false' }}
```

Comment:
Also can't comment on the exact lines, but each instance of `${{ env.DA_ADMIN_EMAIL }}`, and the other `env.` variables can be `$DA_ADMIN_EMAIL`, right? I think they need to be to avoid script injection. Most of the variables are hardcoded and aren't an issue, but `env.CONFIG_ARGS` is user-given.

https://github.com/SuffolkLITLab/ALKiln/pull/843/files#diff-d08e50e93b1ca317bf087718fba26ad50f2d3a37b01eee7241e6d65698be0ebdR127-R144